fedimint-api/src/encoding/mod.rs

```diff
@@ -57,10 +58,18 @@ pub trait Encodable {
     fn consensus_encode<W: std::io::Write>(&self, writer: &mut W) -> Result<usize, std::io::Error>;
 }
 
+// TODO: unify and/or make a newtype?
+pub type ModuleKey = u16;
+
+pub type ModuleRegistry<M> = BTreeMap<ModuleKey, M>;
```

Comment:
I guess the above TODO applies to both types? I think at least the registry could use a newtype.

Reply:
I wrote it when I only had `ModuleKey`, but yes - it crossed my mind that probably both could be a newtype.